e/etcd_bbolt/etcd_bbolt_ubi8.4.sh

```diff
@@ -0,0 +1,75 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : bbolt
+# Version       : v1.3.3
+# Source repo   : https://github.com/etcd-io/bbolt
+# Tested on     : RHEL ubi 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vikas Gupta <vikas.gupta8@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_PATH=go.etcd.io/bbolt
+PACKAGE_NAME=bbolt
+PACKAGE_VERSION=v1.3.3
```

Comment:
Can we make version configurable ?
Even this script is for specific version, looking at the script, I think, build steps could be similar for other versions as well.
So can we make something like :
```
PACKAGE_VERSION=${1:-v1.3.3}
```
So that we can use this script to validate other version of the package as well.

Reply:
Added PACKAGE_VERSION=${1:-v1.3.3}